src/devices/efergy_optical.c

```diff
@@ -21,7 +21,6 @@
 #include "rtl_433.h"
 #include "util.h"
 #include "data.h"
-#include "lib_crc.h"
 
```

Comment:
Do a git remove of the files you added also. It will then look fine when I squash merge the code.